### ISSUE TITLE
Decomplile DLL

### ISSUE THREAD

**Issue by AlexDev404** [AlexDev404/Aurora-v2#3]
Figure out how to decompile DLL

No due date as yet.

**Comment by vinzyYT** [comment]
i dont think it is possible to decompile the dll and use the launcher because when you de-compile something the code isnt allways right

**Comment by AlexDev404** [comment]
@vinzyYT Although the code will not be "always right", there is a possibility that even if there is a whole lot of spaghetti code going all over the place, cleaning up shouldn't be too hard.

**Comment by Milxnor** [comment]
lolol
